networks/suzuka/suzuka-config/src/execution_extension.rs

```diff
@@ -1,11 +1,15 @@
 use godfig::env_default;
 use serde::{Deserialize, Serialize};
 
+/// The execution extension configuration.
+/// This covers Suzuka configurations that do not configure the Maptos executor, but do configure the way it is used.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Config {
+	/// The number of times to retry a block if it fails to execute.
 	#[serde(default = "default_block_retry_count")]
 	pub block_retry_count: u64,
 
+	/// The amount by which to increment the block timestamp if it fails to execute. (This is the most common reason for a block to fail to execute.)
 	#[serde(default = "default_block_retry_increment_microseconds")]
 	pub block_retry_increment_microseconds: u64,
```

Comment:
This could be named as "interval".

Reply:
To me it makes more sense as increment because it is the amount it will increase the timestamp of the block by in order to find an appropriate timestamp.